src/lib/chat/matrix-client.ts

```diff
@@ -36,6 +36,10 @@ export class MatrixClient implements IChatClient {
   private events: RealtimeChatEvents = null;
   private connectionStatus = ConnectionStatus.Disconnected;
 
+  private static readonly RELATES_TO = 'm.relates_to';
+  private static readonly NEW_CONTENT = 'm.new_content';
+  private static readonly REPLACE = 'm.replace';
```

Comment:
a nit but i think we might want to use these values as an `enum` perhaps, otherwise this list can grow quickly :)

Reply:
Ah yes these should have been removed when merging main. They should now be removed :)